### ISSUE TITLE
Punctuation and Language Support

### ISSUE THREAD

**Issue by Schreibmaschine-Berlin** [gestalte-design/zine-machine#3]
Personally I'm looking for the the letters
ö ä ü ß ð þ
but I'm sure many other languages could be supported with only a few more characters.

Additionally some punctuation would be nice
especially the interrobang ‽ because it is a great character

**Comment by gavin-owens** [comment]
Yes, please! I'm excited to expand the language support. We used Helvetica Monospaced to generate the font blocks. If you don't own a license, then perhaps I can add those letters to the file. Not really sure how to share a license for a project like this legally.

**Comment by Schreibmaschine-Berlin** [comment]
I'm thinking maybe Helvetica isn't the most suitable font for this project, especially since it brings up the legal issue of sharing actual character blocks for the purpose of printing. Helvetica is also a tough font to 3d print, looking into it I would need to print the character blocks with a .25mm nozzle to print the thin areas.

I've used Pathway Gothic One as a font for 3D printing since it has a consistent line width and no small inner areas.

**Comment by gavin-owens** [comment]
I agree that a licensed file isn't ideal for an open-source project, but we are going to keep the helvetica monospace as a font set — that I'm pretty sure is legal. 

Add whatever extra fonts you'd like, in this case the more options the better for everyone — maybe just create a new sub folder to organize all the different fonts?
